enhancements/installer/compact-plus-worker.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Compact + Worker deployment as day0 operation
+authors:
+  - "@flaper87"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - TBD
+creation-date: 2022-03-08
+last-updated: 2022-03-08
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Compact + Worker deployment as day0 operation
+
+## Summary
+
+This enhancement proposes to allow for deploying a 4-nodes cluster as a day 0
+operation, where the infrastructure workload can be deployed with HA enabled.
+The 4-nodes cluster would be composed by 3 schedulable master nodes and a
+single worker, which provide users with a highly available cluster.
+
+## Motivation
+
+There are environments where only 4 nodes are available where 4-nodes clusters
+are being deployed. We could serve OpenShift cluster administrators of these
+environments better by allowing them to deploy a 4-nodes cluster as a day0
+operation and make sure their infrastructure workload is highly available.
+
+Today, it is possible to deploy a 4-nodes cluster. The resulting cluster will
```

Comment:
Could you talk some about what services would not be run as HA and what the implications of that are on a cluster with only a single worker node?